setup.py

```diff
@@ -2,10 +2,10 @@
 import os
 
 project_root = os.path.dirname(__file__)
-with open(os.path.join(project_root, "VERSION.txt")) as f:
+with open(os.path.join(project_root, "VERSION.txt"), "r", encoding="UTF-8") as f:
```

Comment:
nit: [Python Unicode docs](https://docs.python.org/3/howto/unicode.html) lowercase `"uft-8"`:

```suggestion
with open(os.path.join(project_root, "VERSION.txt"), "r", encoding="utf-8") as f:
```